remote-method/index.js

```diff
@@ -166,25 +169,29 @@ module.exports = yeoman.generators.Base.extend({
           path: httpPath,
           verb: answers.httpVerb || answers.customHttpVerb
         });
-        this.log('\nLet\'s add another endpoint.');
+        this.log(
+          '\nLet\'s add another endpoint, enter an empty name when done.');
         this.askForEndpoints();
       }.bind(this));
     }.bind(this));
   },
 
   delimAccepts: function() {
-    this.log('Let\'s add some Accept Arguments now.\n');
+    this.log();
+    this.log('Describe the input ("accepts") arguments of your remote method.');
```

Comment:
@bajtos, I think that the brackets combined with the double quotes is a little redundant. I understand why they are both there but I think it would look at little cleaner with `the "accepts" input arguments` or possibly `the input arguments ("accepts" array)` or something along those lines. 

I'm being picky here but I think that the bracket/quote combo draws too much attention.